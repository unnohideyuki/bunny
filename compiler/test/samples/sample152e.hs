main = do
  print $ compare LT LT
  print $ compare LT EQ
  print $ compare LT GT
  print $ compare EQ LT
  print $ compare EQ EQ
  print $ compare EQ GT
  print $ compare GT LT
  print $ compare GT EQ
  print $ compare GT GT
