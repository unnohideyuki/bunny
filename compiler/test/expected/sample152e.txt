EQ
LT
LT
GT
EQ
LT
GT
GT
EQ
